app/views/proposal_mailer/proposal_updated_needs_re_review.html.haml

```diff
@@ -1,7 +1,90 @@
 
-= render partial: "mail_shared/proposal_button",
-  locals: { button_text: t("mailer.proposal_mailer.proposal_updated_needs_re_review.cta") }
+- cta_subheader_foot = t("mailer.proposal_mailer.proposal_updated_needs_re_review.approver_subheader")
+
+%table.container
+  = render partial: "mail_shared/email_header/hero_text", 
+    locals: { text: top_head,
+              style: "no-margin-wrapper" }
+
+  = render partial: "mail_shared/call_to_action/subheader", 
+    locals: { subheader: cta_subheader,
+              vertical: "no-top-margin" }
+  
+  / Start Panel Wrapper
+  = render partial: "mail_shared/panel/header"
+
+  = render partial: "mail_shared/panel/row_description",
+    locals: { icon: panel_icon,
+              action: panel_action,
+              datetime: panel_action_date }
+
+  = render partial: "mail_shared/panel/row_content_modified", 
+    locals: { changes: @changes,
+              last: true}
+
+  = render partial: "mail_shared/call_to_action/subheader", 
+    locals: { subheader: ''}
+
+  = render partial: "mail_shared/call_to_action/button", 
+    locals: { link_path: proposal_url(@proposal), 
+              link_text: proposal_link_text }
+  
+  = render partial: "mail_shared/call_to_action/subheader", 
+  locals: { subheader: '', 
+            height: 'half' }
+
+%hr/
+/ Start Container
+%table.container.contain-attributes
+
+  = render partial: "mail_shared/call_to_action/subheader", 
+    locals: { subheader: panel_preface }
+
+  = render partial: "mail_shared/panel/header_name", 
+                  locals: { name: @proposal.name,
+                            subtext: @proposal.public_id }
+
+  = render partial: "mail_shared/panel/proposal_attributes",
+    locals: { proposal: @proposal }
+
+  - if !@proposal.attachments.nil?
+
+    = render partial: "mail_shared/panel/vertical_buffer"
+
+    - @proposal.attachments.each do |attachment|
+      - attachment_action = t("mailer.attachment_mailer.new_attachment_notification.header",
+                              user_name: attachment.user.full_name,
+                              proposal_name: @proposal.name)
+      - attachment_action_date = attachment.created_at.strftime("%m/%d/%Y") + " at " + attachment.created_at.strftime("%I:%M %P")
+      = render partial: "mail_shared/panel/row_description",
+        locals: { icon: attachment_icon,
+                  action: attachment_action,
+                  datetime: attachment_action_date}
+
+      = render partial: "mail_shared/panel/vertical_buffer"
+      
+  = render partial: "mail_shared/panel/vertical_buffer"
+
+%table.container
+
+  = render partial: "mail_shared/call_to_action/subheader", 
+    locals: { subheader: '' }
+
+  / = render(partial: "mail_shared/call_to_action/email_reply",
+  /   locals: { step: @step, 
+  /             show_step_actions: true, 
+  /             proposal: @proposal })
+  
+  "Button goes here"
```

Comment:
we won't need these buttons since this email goes to oberers / requester so we can remove this placeholder and the commented partial above